### DISCUSSION TITLE
Can field have two columns in element index list?

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#12154] [Q&A] [answered]
Is that possible? let's take for example field that would allow user define date range, between two dates. Would it be possible for field to have two columns in element index list, one with start date and one with end date?

**Comment by brandonkelly** [comment, marked as answer]
Not easily, but the plugin could register additional table columns using the `craft\services\ElementSources::EVENT_DEFINE_SOURCE_TABLE_ATTRIBUTES` event, which is triggered when defining the available table columns for a source.
